StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+Internal.swift

```diff
@@ -404,6 +404,11 @@ extension EmbeddedPaymentElement {
         guard !hasConfirmedIntent else {
             return (.failed(error: PaymentSheetError.embeddedPaymentElementAlreadyConfirmedIntent), nil)
         }
+
+        guard !isUpdating else {
+            return (.failed(error: PaymentSheetError.embeddedPaymentElementConfirmDuringUpdate), nil)
+        }
```

Comment:
can we add a comment for context on why we don't just wait for `update` to finish here.